packages/expo-av/android/src/main/java/expo/modules/av/AVManager.java

```diff
@@ -41,10 +45,18 @@
 import expo.modules.av.video.VideoView;
 import expo.modules.av.video.VideoViewWrapper;
 import expo.modules.interfaces.permissions.Permissions;
+import expo.modules.interfaces.permissions.PermissionsResponseListener;
+
+import com.facebook.react.bridge.ReactContext;
+import com.facebook.react.turbomodule.core.CallInvokerHolderImpl;
 
 import static android.media.MediaRecorder.MEDIA_RECORDER_INFO_MAX_FILESIZE_REACHED;
 
 public class AVManager implements LifecycleEventListener, AudioManager.OnAudioFocusChangeListener, MediaRecorder.OnInfoListener, AVManagerInterface, InternalModule {
+  static {
+    System.loadLibrary("expo-av");
```

Comment:
do you think we can support versioning for this? i've checked existing `System.loadLibrary` in all expo-modules, so far we have expo-gl but it's an unversioned module. well, i think if we want we could support this in versioning. maybe i'd just leave a note here 😅

Reply:
Honestly, I have no idea about versioning this atm 🙄

Reply:
we did support this kind of versioning in react-native but not expo modules. alright, just leave this action item to me. i'll check when i am doing android versioning 😅